<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDialog</class>
 <widget class="QDialog" name="PostDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostDialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>textEdit_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PostDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
